mixsea/overturn.py

```diff
@@ -100,7 +100,8 @@ def eps_overturn(
     dnoise : float, optional
             Noise level of density [kg/m^3] or conservative temperature [°C], depending on overturns_from_CT. Default is 5e-4.
     alpha : float, optional
-            Constant of proportionality between Thorpe and Ozmidov scale. Default is 0.95.
+            Ratio of Ozmidov scale to Thorpe scale (Dillon 1982, Ferron 1998). Default is 0.95.
```

Comment:
Dillon (1982) has alpha=0.8, Ferron et al. (1998) show alpha=0.95. I think we have used 0.95 in the Samoan Passage work since it lined up better with the microstructure observations.